apps/web/src/containers/App/App.tsx

```diff
@@ -187,6 +188,11 @@ const AllRoutes = () => {
             component={Markets}
             routeType="Markets"
```

Comment:
Can you move `/markets` into `/analytics/markets` and then change the route type on both of them to `Analytics`?